lib/Controller/OpenProjectAPIController.php

```diff
@@ -525,4 +525,28 @@ private function base64UrlEncode(string $plainText): string {
 		$base64url = strtr($base64, '+/', '-_');
 		return ($base64url);
 	}
+
+	/**
+	 * check if the group folder set up is already setup or not
+	 *
+	 * @NoCSRFRequired
+	 *
+	 * @return DataResponse
+	 */
+	public function getGroupFolderSetupStatus(): DataResponse {
+		$isGroupFolderSetupCorrect = $this->openprojectAPIService->isGroupFolderSetup();
+		// if everything is ok then we return true and complete the integration
+		if ($isGroupFolderSetupCorrect) {
+			return new DataResponse(
+				[
+					'result' => true
+				]
+			);
+		}
```

Comment:
Maybe can be done like this since we're not reusing the variable anyway
```suggestion
		// if everything is ok then we return true and complete the integration
		if ($this->openprojectAPIService->isGroupFolderSetup()) {
			return new DataResponse(
				[
					'result' => true
				]
			);
		}
```